backend/routers/social_media_marketing_router.py

```
@@ -870,3 +870,179 @@ async def get_campaigns_data(status: Optional[str], platform: Optional[str]) ->
 # Also export as 'router' for compatibility with main.py imports
 router = social_marketing_router
 
+@social_marketing_router.post("/generate-daily-content")
+async def generate_daily_content(
+    request: ContentGenerationRequest = Body(...),
```

Ruff (0.12.2)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)